File: src/classes/api/req_res/update-kmemo-response.ts
'use strict'

import { Kmemo } from '@/classes/datas/kmemo'
import { GkillAPIResponse } from '../gkill-api-response'
import { Kyou } from '@/classes/datas/kyou'

export class UpdateKmemoResponse extends GkillAPIResponse {

    updated_kmemo: Kmemo

    updated_kmemo_kyou: Kyou

    constructor() {
        super()
        this.updated_kmemo = new Kmemo()
        this.updated_kmemo_kyou = new Kyou()
    }

}

File: src/server/gkill/api/req_res/update_kmemo_response.go
package req_res

import (
	"github.com/mt3hr/gkill/src/server/gkill/api/message"
	"github.com/mt3hr/gkill/src/server/gkill/dao/reps"
)

type UpdateKmemoResponse struct {
	Messages message.GkillMessages `json:"messages"`

	Errors message.GkillErrors `json:"errors"`

	UpdatedKmemo *reps.Kmemo `json:"updated_kmemo"`

	UpdatedKyou *reps.Kyou `json:"updated_kyou"`
}
